docs/tutorial/sql.md

```diff
@@ -512,6 +512,9 @@ Sedona provides `SedonaPyDeck` and `SedonaKepler` wrappers, both of which expose
 !!!Note
 	Both SedonaPyDeck and SedonaKepler expect the default geometry order to be lon-lat. If your dataframe has geometries in the lat-lon order, please check out [ST_FlipCoordinates](https://sedona.apache.org/latest-snapshot/api/sql/Function/#st_flipcoordinates)
 
+!!!Note
+	Both SedonaPyDeck and SedonaKepler are designed to work with SedonaDataFrames containing only 1 geometry column. Passing dataframes with multiple geometry columns will cause errors.
```

Comment:
Can you add the following sentence to SedonaPyDeck and SedonaKepler:

To use SedonaPyDeck, GeoPandas and PyDeck must be installed. We recommend the following installation commands:
```
pip install 'pandas<=1.3.5'
pip install 'geopandas<=0.10.2'
pip install pydeck==0.8.0
```

To use SedonaKepler, GeoPandas and KeplerGL must be installed. We recommend the following installation commands:

```
pip install 'pandas<=1.3.5'
pip install 'geopandas<=0.10.2'
pip install keplergl==0.3.2
```

Reply:
Done